web/src/main/webapp/WEB-INF/jsp/team/match/list.jsp

```diff
@@ -0,0 +1,99 @@
+<%@ page contentType="text/html;charset=UTF-8" pageEncoding="utf-8" trimDirectiveWhitespaces="true" session="false" %>
+<%@ taglib tagdir="/WEB-INF/tags" prefix="my" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ taglib prefix="s" uri="http://www.springframework.org/tags" %>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form" %>
+
+<my:pagetemplate title="MatchesOverview">
+    <jsp:attribute name="body">
+         <div class="container">
+             <div class="row">
+                 <div class="col-md-2"></div>
+                 <div class="col-md-8">
+
+                     <h1 class="m-y-3">Matches overview</h1>
+                     <ul class="nav nav-tabs" draggable="true">
+                         <li class="nav-item">
+                             <a href="#played" class="active nav-link" data-toggle="tab">Played</a>
```

Comment:
Taby bych javascriptem nedělal, protože už tak to bude načítat pro tu jednu stránky hodně dat, ktery si uživatel realne nemusi vubec zobrazit a jenom by to žralo výkon. Raději to pls udělej stejně, jak jsem to upravil Denisovi u `team/detail`